client/iOS/Controllers/AboutController.h

```diff
@@ -10,10 +10,17 @@
 
 #import <UIKit/UIKit.h>
 
+#if TARGET_OS_MACCATALYST
```

Comment:
Any reason to use these guards here?
Couldn´t that part just be removed?

Reply:
UIWebView appears to be deprecated and didn't make it into Mac Catalyst at all. Mac Catalyst allows one to compile iOS programs for Mac OS X. When I configured FreeRDP to compile for Mac Catalyst, it failed compiling these files unless I exclude the deprecated portions via these guards.

Reply:
@iiordanov ok, the question is more of a `should this be fixed` one (e.g. `iOS` client has not seen updates for a long time, so this looks more like a general thing to fix)

Reply:
Fair enough. I think it's better that I remove all of the parts that exclude Mac Catalyst and find a way to not include that file in the library when building instead.